lib/calculated_attributes.rb

```diff
@@ -4,16 +4,10 @@
 # Include patches.
 require 'calculated_attributes/rails_patches'
 require 'calculated_attributes/arel_patches'
-raise "Unsupported ActiveRecord version: #{ActiveRecord::VERSION::MAJOR}" unless [3, 4, 5, 6, 7].include? ActiveRecord::VERSION::MAJOR
 
-# Rails 5.2 has its own patches which are different from 5.0. In every other
-# case, just require the patch file for the major version.
-versions = Gem::Version.new(ActiveRecord::VERSION::STRING).canonical_segments.take(2)
-if [[5, 2], [5, 1]].include?(versions)
-  require 'calculated_attributes/rails_5_2_patches'
-else
-  require "calculated_attributes/rails_#{ActiveRecord::VERSION::MAJOR}_patches"
-end
+raise "Unsupported ActiveRecord version: #{ActiveRecord::VERSION::MAJOR}" unless [6, 7].include? ActiveRecord::VERSION::MAJOR
```

Comment:
This is a breaking change for anyone using an older version of Rails. We should bump the [gem version](https://github.com/aha-app/calculated_attributes/blob/DEVOPS-1953-update-gemfile-for-rails7/lib/calculated_attributes/version.rb) from 0.5.0 to 1.0.0 to maintain compatibility with semantic versioning.